source/usage-examples/findOne.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+func main() {
+
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
+	ctx := context.TODO()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Nit: add a newline after each logical block for readability.

Reply:
Thanks for the note. Is it safe to say that going forward, you would prefer that each logical block of code (conditional, interface, loop, function, struct, and scope`{}` ) have a newline after it?

Fix: https://github.com/mongodb/docs-golang/pull/2/commits/8ae2edb9b5239cb859437e4681c40ec94292c4f8

Reply:
It's not wrong, but I wouldn't necessarily put a newline between connecting and deferring a disconnect, because I think of those as the same task- every connected client should have a deferred disconnect- but the way I think about it every task should get its own block. Like below, where creating the FindOne options, using them in a FindOne, and checking the error are together, because it's the same task, but they're separated from other tasks by newlines.
What constitutes different tasks can be somewhat subjective though (if it's not clear from this conversation). Feel free to ask follow up questions if I'm not being clear.

Reply:
Thanks for clarifying Isabella and Kevin. I thought "logical block" was referring to certain go language features rather than logical units of work. I will go through and correct the edits from https://github.com/mongodb/docs-golang/commit/8ae2edb9b5239cb859437e4681c40ec94292c4f8.